test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs

```diff
@@ -171,7 +171,7 @@ public async Task GivenAnInvalidEtag_WhenPatching_ThenAnErrorShouldBeReturned()
                 patchDocument,
                 "5"));
 
-            Assert.Equal(HttpStatusCode.BadRequest, exception.Response.StatusCode);
```

Comment:
>BadRequest [](http://example.com/codeflow?start=40&length=10)

why did you changed?

Reply:
Because weaketag wasn't working and test was failing in applying patch to deceasedDateTime which is field and not array, and instead of add operation replace operation should be used.